src/status_im/ui/screens/screens.cljs

```diff
@@ -565,7 +565,7 @@
             :component network-info/network-info}
            {:name      :rpc-usage-info
             :options   {:topBar {:title {:text (i18n/label :t/rpc-usage-info)}}}
-            :component rpc-usage-info/usage-info}
+            :component rpc-usage-info/usage-info-container}
```

Comment:
its better to keep this name for container and rename usage-info to usage-info-render

Reply:
so here we should keep rpc-usage-info/usage-info

Reply:
I initially started with that naming convention but I looked at other files where it follows as above so I followed the same (doesn't want to deviate from the code ambience). I can change the name if needed. Kindly advice. @flexsurfer